### ISSUE TITLE
Can ErrorBoard accept any custom parameter to provide better filter and grouping?

### ISSUE THREAD

**Issue by kumarvikram** [Lapple/ErrorBoard#27]
Let me tell you my current scenario. We have our application deployed on multiple servers and they all are using the same instance(same url and port) of ErrorBoard. 
For any new feature, we release first to one or two servers and then after QA we release it to all servers. So, it is difficult to track any error occurred on a newly updated server.
Just wondering whether I can pass a custom parameter like server name using ErrorBoard code snippet and apply filter on that in dashboard.

**Comment by Lapple** [comment]
Yes, that feature makes a lot of sense. We can add some sort of metadata support for error entry. Do you think a single string is enough? In parallel, I will see how to implement it better.

**Comment by Lapple** [comment]
I did some work in 39e381a to bc24c3e, please check how it fits your use case.

**Comment by toohamster** [comment]
why cannot catch ajax error, net error ?
